content/rs/release-notes/rs-7-2-releases/_index.md

```diff
@@ -13,8 +13,28 @@ categories: ["RS"]
 ​[​Redis Enterprise Software version 7.2](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
 
 This version offers:
+ 
+- Redis 7.0 and 7.2 features
 
-- TBA
+- Three Redis database versions: 7.2, 6.2, 6.0
+
+- Auto Tiering (enhanced Redis on Flash with Speedb) and license updates
+
+- Redis ACL selectors and enhanced key-based permissions
+
+- RESP3 support
+
+- Sharded pub/sub
+
+- A preview of the redesigned Cluster Manager UI (admin console)
+
+- New INFO fields
+
+- Log rotation enhancements
+
+- Triggers and functions preview
+
+- Large module support
```

Comment:
@maayanagranat - I think that customer facing this name has no meaning. The later description talked about the ability to upgrade clusters containing databases with modules in combination with with Operating System (OS) upgrades. We should focus on that part
@rrelledge fyi

Reply:
Updated